modules/dlm/src/main/java/org/elasticsearch/dlm/DataLifecycleService.java

```diff
@@ -360,9 +360,9 @@ public void onFailure(Exception e) {
         }
     }
 
-    private RolloverRequest getDefaultRolloverRequest(String dataStream) {
+    private RolloverRequest getDefaultRolloverRequest(String dataStream, TimeValue dataRetention) {
```

Comment:
Shall we make this static/default package visible and unit test it? (to make sure it's using the `RolloverConfiguration` stuff properly?)

Reply:
Yes, good catch.